nrt/data/IEEE57/RegulationProportional/IEEE57_1_StepLoad/IEEE57.par

```diff
@@ -0,0 +1,681 @@
+    <par type="DOUBLE" name="generator_RTfPu" value="0.0"/>
+    <par type="DOUBLE" name="generator_XTfPu" value="0.1"/>
+    <par type="DOUBLE" name="governor_KGover" value="5"/>
+    <par type="DOUBLE" name="governor_PMin" value="0"/>
+    <par type="DOUBLE" name="governor_PMax" value="1485"/>
+    <par type="DOUBLE" name="governor_PNom" value="1485"/>
+    <par type="DOUBLE" name="Pm_ValueIn" value="0"/>
+    <par type="DOUBLE" name="URef_ValueIn" value="0"/>
+    <par type="DOUBLE" name="voltageRegulator_EfdMinPu" value="-5"/>
+    <par type="DOUBLE" name="voltageRegulator_EfdMaxPu" value="5"/>
+    <par type="DOUBLE" name="voltageRegulator_Gain" value="20"/>
+    <par type="DOUBLE" name="voltageRegulator_LagEfdMin" value="0"/>
+    <par type="DOUBLE" name="voltageRegulator_LagEfdMax" value="0"/>
+    <reference name="generator_P0Pu" origData="IIDM" origName="p_pu" type="DOUBLE"/>
+    <reference name="generator_Q0Pu" origData="IIDM" origName="q_pu" type="DOUBLE"/>
+    <reference name="generator_U0Pu" origData="IIDM" origName="v_pu" type="DOUBLE"/>
+    <reference name="generator_UPhase0" origData="IIDM" origName="angle_pu" type="DOUBLE"/>
+  </set>
+  <set id="50">
+    <par type="INT" name="nbGen" value="7"/>
+    <par type="DOUBLE" name="weight_gen_0" value="6048"/>
+    <par type="DOUBLE" name="weight_gen_1" value="9281.25"/>
+    <par type="DOUBLE" name="weight_gen_2" value="8755.2"/>
+    <par type="DOUBLE" name="weight_gen_3" value="8755.2"/>
+    <par type="DOUBLE" name="weight_gen_4" value="6539.4"/>
+    <par type="DOUBLE" name="weight_gen_5" value="6539.4"/>
+    <par type="DOUBLE" name="weight_gen_6" value="9281.25"/>
+  </set>
+  <set id="51">
+    <par type="DOUBLE" name="CAPACITOR_no_reclosing_delay" value="300"/>
+    <par type="DOUBLE" name="DANGLING_LINE_currentLimit_maxTimeOperation" value="90"/>
+    <par type="DOUBLE" name="LINE_currentLimit_maxTimeOperation" value="90"/>
+    <par type="DOUBLE" name="LOAD_Tp" value="90"/>
+    <par type="DOUBLE" name="LOAD_Tq" value="90"/>
+    <par type="DOUBLE" name="LOAD_alpha" value="1"/>
+    <par type="DOUBLE" name="LOAD_alphaLong" value="0"/>
+    <par type="DOUBLE" name="LOAD_beta" value="2"/>
+    <par type="DOUBLE" name="LOAD_betaLong" value="0"/>
+    <par type="BOOL" name="LOAD_isControllable" value="false"/>
+    <par type="BOOL" name="LOAD_isRestorative" value="false"/>
+    <par type="DOUBLE" name="LOAD_zPMax" value="100"/>
+    <par type="DOUBLE" name="LOAD_zQMax" value="100"/>
+    <par type="DOUBLE" name="REACTANCE_no_reclosing_delay" value="0"/>
+    <par type="DOUBLE" name="TFO_currentLimit_maxTimeOperation" value="90"/>
+    <par type="DOUBLE" name="TFO_t1st_HT" value="60"/>
+    <par type="DOUBLE" name="TFO_t1st_THT" value="30"/>
+    <par type="DOUBLE" name="TFO_tNext_HT" value="10"/>
+    <par type="DOUBLE" name="TFO_tNext_THT" value="10"/>
+    <par type="DOUBLE" name="TFO_tolV" value="0.014999999700000001"/>
+  </set>
+  <set id="52">
```

Comment:
Set not used because: event params are in event.par

Reply:
Deleted